tracker_test.go

```diff
@@ -18,6 +19,7 @@ func TestCreateTracker(t *testing.T) {
 	}
 
 	_, created := test.CreateTrackerCreated(t, nil, nil, &controller, &payload)
+	defer gormsupport.DeleteCreatedEntities(DB)()
```

Comment:
This function registers itself with gorm to track the different entities that are created in the DB. You can't initialize it _After_ you've created the entities and expect it to go back in time and track them...

Place all of these statements right after the require.Resource(DB) stmt.